concrete/src/File/Image/Thumbnail/Type/Version.php

```diff
@@ -14,14 +14,16 @@ class Version
     protected $directoryName;
     protected $handle;
     protected $name;
+    protected $sizingMode;
     protected $width;
     protected $height;
     protected $isDoubledVersion;
 
-    public function __construct($directoryName, $handle, $name, $width, $height, $isDoubledVersion = false)
+    public function __construct($directoryName, $handle, $name, $sizingMode, $width, $height, $isDoubledVersion = false)
```

Comment:
I just figured out BC. I suppose you mean Backward Compatible? Duh!!! Sorry, not a native English speaker. My brain has to work hard to get acronyms...

Anyway, I think my last modification deals with that